### PR TITLE
Add katex support

### PR DESCRIPTION
**Pull request by LitangDingzhen**
related #71 

I added a plugin to markdown-it so that it displays latex formulas, and here are the tests:

image

image

### CODE REVIEW

**Comment by wong2** [comment]
nice job

File: src/manifest.json [hunk 1]
```diff
@@ -13,12 +13,19 @@
...
       "include_globs": ["*.google.*/*"],
       "js": ["content-script.js"],
-      "css": ["github-markdown.css", "styles.css"]
+      "css": ["github-markdown.css", "styles.css", "katex.css"]
+    }
+  ],
+  "web_accessible_resources": [
+    {
+      "resources": ["fonts/*"],
+      "matches": ["https://*/*"]
```
Comment (wong2) [inline src/manifest.json:28]: Why is this necessary?

File: src/styles.css [hunk 2]
```diff
@@ -59,5 +59,5 @@
 
 .chat-gpt-container #answer code {
   white-space: pre-wrap;
-  word-break: break-all;
+  word-break: break-word;
```
Comment (wong2) [inline src/styles.css:62]: Why?

File: src/manifest.json (hunk 1 above)
Comment (LitangDingzhen) [reply src/manifest.json:28]: I added these fonts in the plugin so that I don't have to visit google. If I don't import it directly, it will give me an error:

image

It can also be introduced through a CDN, but that will be slower

File: src/styles.css (hunk 2 above)
Comment (LitangDingzhen) [reply src/styles.css:62]: > Why?

'break-all' breaks words, so it might be nicer to use 'break-word'

Comment (LitangDingzhen) [reply src/styles.css:62]: > Why?

Here's an example

with `break-all`

image

with `break-word`

image

File: src/manifest.json (hunk 1 above)
Comment (wong2) [reply src/manifest.json:28]: got it

**Comment by wong2** [comment]
I'm still considering whether to accept this change, as it increase the installation package from 95K to 1.2M, but not many users will benefit from this feature.

**Comment by LitangDingzhen** [comment]
> I'm still considering whether to accept this change, as it increase the installation package from 95K to 1.2M, but not many users will benefit from this feature.

I optimized the bundle size and now the bundle size is 492KB.
